Romuashi/hometask-2/get_middle_charachter.rb

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+def get_middle(str)
+  x = str.length / 2
```

Comment:
try to avoid meaningless names of variables, better to give some descriptive name for example `middle_position` or something similar